framework/src/application/network/schema.ts

```diff
@@ -32,7 +31,6 @@ export const customNodeInfoSchema = {
 };
 
 export const customPeerInfoSchema = {
-	...p2pSchemas.peerInfoSchema,
 	$id: '/peerInfo/custom',
 	type: 'object',
 	properties: {
```

Comment:
shouldn't the line below also suppose to be deleted? 
```
...p2pSchemas.peerInfoSchema.properties,
```

Reply:
true, i missed it

Reply:
addressed